src/riak_cs_s3_auth.erl

```diff
@@ -43,14 +43,16 @@
 identify(RD,_Ctx) ->
```

Comment:
The spec have changed to 
`-spec identify(#reqdata{}, #context{}) -> {string() | undefined, string() | {v4, proplists:proplist()}.` ? Can we narrow down from the behaviour callback definition?